homework/homework.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"fmt"
+	"gocourse05/core/camera"
+	"math/rand/v2"
+)
+
+type ID interface {
+	GetId() int
+}
+
+type Metadata interface {
+	GetMetaData() string
```

Comment:
https://go.dev/doc/effective_go#Getters

```suggestion
	MetaData() string
```